website/pages/docs/plugins/destinations/azblob/overview.md

```diff
@@ -0,0 +1,55 @@
+# Azure Blob Storage Destination Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+
+<Badge text={"Latest: " + getLatestVersion("destination", "azblob")}/>
+
+This destination plugin lets you sync data from a CloudQuery source to remote Azure Blob Storage storage in various formats such as CSV, JSON.
+
+This is useful in various use-cases, especially in data lakes where you can query the data direct from Athena or load it to various data warehouses such as BigQuery, RedShift, Snowflake and others.
```

Comment:
```suggestion
```

Let's remove this paragraph for now, as it's not applicable. We could talk about https://azure.microsoft.com/en-gb/products/data-lake-analytics/ but not sure if that will actually work, we should test it first